ui/components/Button.tsx

```diff
@@ -18,8 +18,8 @@ export interface Props extends ButtonProps {
 function UnstyledButton({ loading, ...props }: Props) {
   return (
     <MaterialButton
-      variant="outlined"
-      color="primary"
+      variant={props.variant || "outlined"}
+      color={props.color || "primary"}
```

Comment:
Could pull this out into `const defaultProps` maybe?

Reply:
Updated